reports/Colorado.md

```diff
@@ -107,6 +107,18 @@ id: co-denver-7
 * https://youtu.be/xNLZ7exVxq4?t=110
 
 
+### Police shoot bystander in eye | May 30th
+
+Law enforcement officer riding on the back of a Denver Police Department truck fired a projectile at bystander's face without warning and blinded him in one eye.
```

Comment:
The attached links do not show the incident, only the injured man's claims of what happened.  From this evidence we don't know what happened and don't have any evidence the claim is true.

Might be better to state: "Man claims law enforcement officer riding on the back of a Denver Police Department truck fired a projectile at him without warning, blinding him in one eye."

Reply:
Good point, made the update.